src/gsy_e/gsy_e_core/simulation/simulation.py

```diff
@@ -438,10 +448,24 @@ def _execute_simulation(
             self.progress_info.update(slot_no, slot_count, self._time, self.config)
 
             self.area.cycle_coefficients_trading(self.progress_info.current_slot_time)
+            log.error(self.area.current_market_time_slot)
```

Comment:
I think that this can be removed, right ?

Reply:
yes, remnant of testing (normally I use print() which is why this got overlooked).

Reply:
I fixed this in here [0d2cbfc](https://github.com/gridsingularity/gsy-e/pull/1559/commits/0d2cbfcb410817f89992821aed22687ae80b80af)